versions.tf

```
@@ -3,7 +3,7 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = ">= 3.36, !=4.0.0, !=4.1.0, !=4.2.0, !=4.3.0, !=4.4.0, !=4.5.0, !=4.6.0, !=4.7.0, !=4.8.0"
+      version = ">= 4.0.0"
```

There were some s3 changes that they reverted in the low 4 versions which is why it's like this. We're probably compatible with these changes regardless, but it may be worth being a bit more strict in saying that we should only allow above 4.9.0 to get past those versions.
```suggestion
      version = ">= 4.9.0"
```

We may also want to put an upper limit for when there's inevitably a breaking change in the provider:
```suggestion
      version = ">=4.9.0, <5.0.0"
```